This patch makes svlogd log in localtime rather than UTC when passed -tt

	Vincent Danen (vdanen@annvix.org)

--- runit-1.3.1/src/fmt_ptime.c.avx	2005-08-30 01:04:10.000000000 -0400
+++ runit-1.3.1/src/fmt_ptime.c	2005-08-30 01:04:10.000000000 -0400
@@ -8,7 +8,7 @@
 
   if (ta->sec.x < 4611686018427387914ULL) return(0); /* impossible? */
   u =ta->sec.x -4611686018427387914ULL;
-  if (! (t =gmtime((time_t*)&u))) return(0);
+  if (! (t =localtime((time_t*)&u))) return(0);
   fmt_ulong(s, 1900 +t->tm_year);
   s[4] ='-'; fmt_uint0(&s[5], t->tm_mon +1, 2);
   s[7] ='-'; fmt_uint0(&s[8], t->tm_mday, 2);
